src/services/ui/e2e/tests/home/index.spec.ts

```diff
@@ -1,21 +1,37 @@
 import { test, expect } from "@playwright/test";
+import { chromium } from "@playwright/test";
 import * as $ from "@/selectors";
 
+const user = {
+  userName: "george@example.com",
+  password: "bigTUNA1!"
+};
+
 test("has title", async ({ page }) => {
-  await page.goto("/");
+  await page.goto("https://mako-dev.cms.gov/");
 
   // Expect a title "to contain" a substring.
-  await expect(page).toHaveTitle(/CMS MAKO/);
+  await expect(page).toHaveTitle(/.*CMS MAKO/);
 });
 
-test("get issues link", async ({ page }) => {
-  const navSelectors = new $.NavSelectors(page);
-  await page.goto("/");
-
-  // Click the issues link.
-  await navSelectors.issuesDropDown.click();
-  await navSelectors.allIssuesLink.click();
+test("has faq Page", async ({ page }) => {
+  await page.goto("https://mako-dev.cms.gov/");
+  await page.getByRole("link", { name: "FAQ" }).click();
+});
 
-  // Expects the URL to contain intro.
-  await expect(page).toHaveURL(/.*issues/);
+test("log in test", async ({ page }) => {
+  //test.setTimeout(10000);
+  const loggedInIndicator = await page.getByRole("button", { name: "submit" });
+  if (loggedInIndicator) {
+    await page.goto("https://mako-dev.cms.gov/");
+    await page.getByRole("button", { name: "Sign In" }).click();
+    await page.getByRole("textbox", { name: "name@host.com" }).click();
+    await page.getByRole("textbox", { name: "Password" }).click();
+    await page.getByRole("textbox", { name: "Password" }).click();
+    await page.getByRole("button", { name: "submit" }).click();
+    console.log("User successfully logged in!");
```

Comment:
I think we need to bake in some logic here to address the tickets criteria: 

`auth/unauth access.   If logged in, you should see this.  If not logged in, yhou should see this.`

`fine grain auth access.  If Virginia, you should see only Virginia.  If North Dakota and South Dakota, you should see only the Dakotas.`